modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -65,7 +65,7 @@ impl RecoveryFromHistory for MintRecovery {
             MintRecovery {
                 state: MintRecoveryState::from_backup(
                     snapshot,
-                    30,
+                    200,
```

Comment:
To start recovery we need to upfront derive this many * num_tiers of nonces, which was actually somewhat substantial. I'm planning to increase it to 100.

Reply:
Already fixed as part of #6084